src/templates/src/index.ts.squirrelly

```diff
@@ -1,5 +1,17 @@
-import { whisper, {{ it.aptitudes.join(", ") }} } from "@oliveai/ldk";
```

Comment:
It became needlessly complex to only add the imports used for the aptitude examples selected, so I just import all of them here (and the only bad thing is that there are unused examples).